### ISSUE TITLE
Error when creating new mumc_config.yaml by converting from mumc_config.py

### ISSUE THREAD

**Issue by keppo070** [terrelsa13/media_cleaner#96]
There is no version in the mumc.py but I used this  ```sudo git clone https://github.com/terrelsa13/MUMC.git```

```
WARNING: One or more config variable(s) were missing from mumc.py.
 MUMC has assumed defaults values for the missing variable(s).
 Please be sure to review these variable(s) in /home/patrick/programmeren/python/mumc5b/MUMC/mumc_config.yaml

LegacyConfigNameWarning: The 'favorited_advanced_tv_library_genre' variable is missing from mumc.py
LegacyConfigNameWarning: The 'print_movie_post_processing_info' variable is missing from mumc.py
LegacyConfigNameWarning: The 'print_episode_post_processing_info' variable is missing from mumc.py
LegacyConfigNameWarning: The 'print_audio_post_processing_info' variable is missing from mumc.py
LegacyConfigNameWarning: The 'script_header_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'script_warnings_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'user_header_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'movie_delete_info_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'movie_keep_info_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'episode_delete_info_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'episode_keep_info_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'audio_delete_info_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'audio_keep_info_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'movie_post_processing_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'episode_post_processing_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'audio_post_processing_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'summary_header_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'movie_summary_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'episode_summary_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'audio_summary_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'script_footer_format' variable is missing from mumc.py

Traceback (most recent call last):
  File "/home/patrick/programmeren/python/mumc5b/MUMC/mumc.py", line 105, in <module>
    MUMC()
  File "/home/patrick/programmeren/python/mumc5b/MUMC/mumc.py", line 27, in MUMC
    cfg,init_dict=importConfig(init_dict,cmdopt_dict)
  File "/home/patrick/programmeren/python/mumc5b/MUMC/mumc_modules/mumc_config_import.py", line 98, in importConfig
    convert_legacyConfigToYAML(legacy_dict,init_dict['mumc_path'],init_dict['config_file_name_no_ext'])
  File "/home/patrick/programmeren/python/mumc5b/MUMC/mumc_modules/mumc_config_convert.py", line 59, in convert_legacyConfigToYAML
    failvar=failvar
UnboundLocalError: local variable 'failvar' referenced before assignment

```

**Comment by terrelsa13** [comment]
Yep. I made a mistake. Fixed it with the [latest v5.11](https://github.com/terrelsa13/MUMC).

**Comment by terrelsa13** [comment]
The version is printed to the console output when the script is run.
The version in code resides in ```../mumc_modules/mumc_versions.py```.

Let me know if the script is still giving you issues.

**Comment by keppo070** [comment]
```
WARNING: One or more config variable(s) were missing from mumc.py.
 MUMC has assumed defaults values for the missing variable(s).
 Please be sure to review these variable(s) in /home/patrick/programmeren/python/mumc5b/MUMC/mumc_config.yaml

LegacyConfigNameWarning: The 'favorited_advanced_tv_library_genre' variable is missing from mumc.py
LegacyConfigNameWarning: The 'print_movie_post_processing_info' variable is missing from mumc.py
LegacyConfigNameWarning: The 'print_episode_post_processing_info' variable is missing from mumc.py
LegacyConfigNameWarning: The 'print_audio_post_processing_info' variable is missing from mumc.py
LegacyConfigNameWarning: The 'script_header_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'script_warnings_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'user_header_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'movie_delete_info_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'movie_keep_info_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'episode_delete_info_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'episode_keep_info_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'audio_delete_info_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'audio_keep_info_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'movie_post_processing_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'episode_post_processing_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'audio_post_processing_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'summary_header_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'movie_summary_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'episode_summary_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'audio_summary_format' variable is missing from mumc.py
LegacyConfigNameWarning: The 'script_footer_format' variable is missing from mumc.py


ConfigValueError: The same tag cannot be used for both advanced_settings > blacktags and advanced_settings > whitetags
        To proceed the following tags need to be fixed:: ['']
```
I had to change 
```
  whitetags:
  - ''
  blacktags:
  - ''
```
into

```
  whitetags: []
  blacktags: []
```

Not sure if this is an issue you want to address. Just commenting. Fixed for me

**Comment by terrelsa13** [comment]
Good find! This is fixed in [v5.3.12](https://github.com/terrelsa13/MUMC/releases). The legacy config converter should now populate an empty list for ```mumc_config.yaml``` when the legacy config has an empty string (i.e ```''```).
